atom/browser/api/atom_api_system_preferences.h

```diff
@@ -93,6 +100,10 @@ class SystemPreferences : public mate::EventEmitter<SystemPreferences> {
   HWND window_;
 
   std::string current_color_;
+
+  bool invertered_color_scheme_ = false;
```

Comment:
`invertered_color_scheme_` should be initialized with `IsInvertedColorScheme()`.